Software/C/GoPiGo3.cpp

```diff
@@ -9,7 +9,68 @@
  *  C++ drivers for the GoPiGo3
  */
 
-#include "GoPiGo3.h"
+#include <GoPiGo3.h>
+#include <stdexcept>
+#include <fcntl.h>
+#include <sys/ioctl.h>
+#include <linux/spi/spidev.h>
+#include <stdio.h>            // for printf
+#include <string.h>           // for strstr
+#include <sys/time.h>         // for clock_gettime
+#include <unistd.h>
+
+#define FIRMWARE_VERSION_REQUIRED "0.3." // Firmware version needs to start with this
```

Comment:
This definition should be left in the .h file. Same applies to the following definitions on lines 24 and 26.